### ISSUE TITLE
segfaults with changes from 1.11.5->1.11.6

### ISSUE THREAD

**Issue by stonier** [ros/ros_comm#469]
I'm seeing segfaults when starting roscore on a shiny new installation of trusty with indigo from the ros public repo (i.e. not shadow-fixed).
- rosmaster on its own is fine
- roscore -> rosout is segfaulting and respawning
- romaster + rosout -> rosout is segfaulting
- rosmaster + talker -> talker is segfaulting with the same backtrace (below)

I apt-get removed `ros-indigo-*` scp'd across the slightly older `1.11.5` installation from my work computer and that runs without a problem so it looks like it's in the `1.11.5->1.11.6` upgrade.

The backtrace:

```
Starting program: /opt/ros/indigo/lib/rosout/rosout rosout
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGSEGV, Segmentation fault.
0x00007ffff7b41cc7 in ros::Transport::Transport() () from /opt/ros/indigo/lib/libroscpp.so
(gdb) backtrace
#0  0x00007ffff7b41cc7 in ros::Transport::Transport() ()
   from /opt/ros/indigo/lib/libroscpp.so
#1  0x00007ffff7b4d1e0 in ros::TransportTCP::TransportTCP(ros::PollSet*, int) ()
   from /opt/ros/indigo/lib/libroscpp.so
#2  0x00007ffff7b3c828 in ros::ConnectionManager::start() ()
   from /opt/ros/indigo/lib/libroscpp.so
#3  0x00007ffff7b5f049 in ros::start() () from /opt/ros/indigo/lib/libroscpp.so
#4  0x00007ffff7b39d51 in ros::NodeHandle::construct(std::string const&, bool) ()
   from /opt/ros/indigo/lib/libroscpp.so
#5  0x00007ffff7b3a03f in ros::NodeHandle::NodeHandle(std::string const&, std::map<std::string, std::string, std::less<std::string>, std::allocator<std::pair<std::string const, std::string> > > const&) () from /opt/ros/indigo/lib/libroscpp.so
#6  0x0000000000407ca1 in main ()
```

**Comment by dirk-thomas** [comment]
Can you please check if that is already addressed with #465?

**Comment by dirk-thomas** [comment]
I will close this for now since I expect it to be fixed with the referenced fix which has been released as version 1.11.7.

If the problem still exists with the new release please comment again to reopen the ticket. If you confirmed that the problem is solved a comment would be appreciated as well.

**Comment by stonier** [comment]
:+1: all good with 1.11.7.
